view/frontend/web/js/instantsearch.js

```diff
@@ -755,33 +599,224 @@ define([
                     });
                 });
             }
-    
-            var isStarted = false;
-    
-            function startInstantSearch() {
-                if (isStarted === true) {
-                    return;
-                }
-    
-                search = algoliaCommon.triggerHooks(
+
+            this.startInstantSearch(search, mockAlgoliaBundle);
+
+            this.addMobileRefinementsToggle();
+        },
+
+        /**
+         * @deprecated - these hooks will be removed in a future version
```

Comment:
Should it happen in 3.16 ?

Reply:
Yeah probably - they've been there forever it seems...